build/verify-test-urls.sh

```diff
@@ -1,70 +0,0 @@
-#! /bin/bash
-
-# Automatically exit on error
-set -e
-
-ANNOTATIONS=$(mktemp)
-ERRORS=0
-
-# Keep short because GitHub doesn't wrap it
-ERROR_MESSAGE="Every new or edited feature must be tested manually before merging.
-To help testing, we're progressively adding test URLs in each feature.
-
-You can find or create a test URL:
-
-- on our sandbox repo: https://github.com/refined-github/sandbox
-- on previous PRs for this feature
-
-The section must be appended to each .tsx file:
-
-/*
```

Comment:
I need to restore this message too, to help user document new features. The message needs to be updated.